syntax = "proto3";

message RGBValue {
  int32 red = 1;
  int32 green = 2;
  int32 blue = 3;
}

message LightRequest {
  int32 lights = 1;
  int32 offset = 2;
  repeated RGBValue value_array = 3;
}

message SensorLog {
  int32 pot = 1;
}
